js/10-base.js

```diff
@@ -93,6 +93,17 @@ Liberapay.init = function() {
         ).prop('selected', true);
         history.pushState(null, null, location.pathname + href + location.hash);
     });
+    
+    $('input[data-required-if-checked]').each(function(index, value) {
+        var radio = $(value).attr('data-required-if-checked');
+        $($(radio).parents('form')).on('click', function(e) {
+            if($(radio).is(':checked')) {
+                $(value).prop('required', true);
+            } else {
+                $(value).prop('required', false);
+            }
```

Comment:
This `if … else …` block can be simplified to `$(value).prop('required', $(radio).is(':checked'));`.